future_challenges_for_RSEng.tex

```diff
@@ -130,24 +134,32 @@ \section{Introduction}
 with increasing integration into various aspects of research.
 In this opinion paper, we try to imagine the state of our field
 and community in the mid-term future, focusing on future challenges.
+While future projections are common in industry \cite{gartner}, so far there is a projection lacking for the RSEng space.
 We present here the results of a workshop discussion at the
 deRSE25 conference in Karlsruhe\cite{Goth2025EndRSEng}, without
 claiming to represent the entire deRSE community.
 Nevertheless, by disseminating the discussed ideas,
 we aim to engage the community in creating a future worth living
-and advance the conversation about how Research Software Engineering should, or could, develop.
-In the following sections, we consider a number of areas:
-the evolution of the RSE definition;
-the development of RSE education;
-the growth of the RSE communities;
-the impact of automation and generative artificial intelligence in RSE work;
-the emerging ethical considerations;
-the transformations expected in research organisations;
-and, finally, if there will be other areas in academia covered by new groups of RTPs (\emph{Research Technical Professional}s).
+and advance the conversation about how Research Software Engineering should, or could, develop.\\
+It is already difficult to structure the spectrum of RSE tasks and responsibilities today given the spectrum of research software \cite{hasselbring2024}
+and it is even more challenging to structure the aspects that will become relevant for the RSE community in the future.
+Nevertheless, we decided to consider:
+\begin{itemize}
+\item The evolution of the term RSE and its definition
+\item Growth and development of the RSE community
+\item The impact of automation and generative artificial intelligence in RSE work
+\item The impact of complexity and usability in research software
+\item Ethics and social considerations
+\item The transformations expected in academic institutions
+\item Impact of alternate career paths for RSEs and other RTPs(\emph{Research Technical Professional}s)
+\end{itemize}
```

Comment:
That's a matter of taste, but I associate bullet lists (especially with this large space between items) with drafts. I find the inline style (as before) more expected in an introduction.

This could also make the introduction shorter, helping with pagination.

What do you think?

Reply:
done in latest commit.